packages/expo-web-browser/ios/EXWebBrowser/EXWebBrowser.m

```diff
@@ -105,13 +105,14 @@ - (dispatch_queue_t)methodQueue
     safariVC.preferredControlTintColor = [EXWebBrowser convertHexColorString:arguments[WebBrowserControlsColorKey]];
   }
   safariVC.delegate = self;
+  // By setting the modal presentation style to OverFullScreen, we disable the "Swipe to dismiss"
+  // gesture that is causing a bug where sometimes `safariViewControllerDidFinish` is not called.
+  // There are bugs filed already about it on OpenRadar.
+  [safariVC setModalPresentationStyle: UIModalPresentationOverFullScreen];
```

Comment:
This reverts https://github.com/expo/expo/commit/3f901cf5ca6255d46b89808ceb431136e228945d, which fixed fullscreen browser being presented in an iOS 13 modal. Could you please verify that this no longer happens? Or maybe we need to set this `modalPresentationStyle` on both `safariVC` and `safariHackVC`?

Reply:
Yes. The browser is presented fullscreen. We need to set this style on both VC

Reply:
Ah, ok, sorry, now I see that the style is set on both VCs. 👍

Reply:
No problem 😊Thank you